dali/base/dafdesc.cpp

```diff
@@ -2338,7 +2338,34 @@ IFileDescriptor *createFileDescriptor(const char *lname,IGroup *grp,IPropertyTre
     return res;
 }
 
+IFileDescriptor *createFileDescriptor(const char *lname, const char *userId, const char *jobId,
```

Comment:
minor: probably userid and jobId shouldn't be part of this function, but set outside on return of the create IFileDescriptor.

Because they are arbitrary properties as far as IFileDescriptor is concerned..

Reply:
ok